src/right-field/results-table/ResultsTable.css

```diff
@@ -1,7 +1,7 @@
 
 .results-table{
-    display: inline-block;
-    position: relative;
+    display: flex;
+    justify-content: center;
```

Comment:
This is for the text in the table, right? In that case, don't center the text. This reduces the readability of the text.

Reply:
It is not for the text in the tables, but I have changed it to not center the text in the table in [commit c0fbb5d](https://github.com/SolidLabResearch/generic-data-viewer/pull/24/commits/c0fbb5def8863ac0e129bb452c6811af85a3dc40)